### PR TITLE
docs(upgrade): update documentation on README.md , index.md

### LINKED ISSUE #191: docs: Improve and update documentation content

### PR DESCRIPTION
**Pull request by Satarupa22-SD**
## Pull Request description

The scicookie templates aims to streamline the creation process by eliminating boilerplate structures. However, the project description was mistakenly categorized as boilerplate content. This pull request aims to rectify this misunderstanding and provide clarity regarding the project's description. 
Resolves #191

## Pull Request checklists

This PR is a:

- [x] bug-fix
- [ ] new feature
- [x] maintenance

About this PR:

- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [x] pre-commit hooks were executed locally.
- [x] this PR requires a project documentation update.

Author's checklist:

- [x] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more
      complexity.
- [x] New and old tests passed locally.

### CODE REVIEW

**Comment by Satarupa22-SD** [comment]
@xmnlab

File: README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
 project templates and is based on
 [Cookieninja](https://github.com/cookieninja-generator/cookieninja) **A
-Cookiecutter Fork** command-line utility. It serves as a boilerplate which can
-be used by beginners as well as full fledged developers to simplify the project
-creation process and save considerable amount of time. Cookieninja enables
-projects with an initial layout that includes recommended tools, workflows, and
-project structure.
+Cookiecutter Fork** command-line utility. It serves as an initial structure
+which can be used by beginners as well as full-fledged developers to simplify
+the project creation process and save considerable amount of time. Cookieninja
```
Comment (xmnlab) [inline README.md:15]:
```suggestion
the project creation process and save considerable amount of time. SciCookie
```

File: docs/index.md [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
 project templates and is based on
 [Cookieninja](https://github.com/cookieninja-generator/cookieninja) **A
-Cookiecutter Fork** command-line utility. It serves as a boilerplate which can
-be used by beginners as well as full fledged developers to simplify the project
-creation process and save considerable amount of time. Cookieninja enables
-projects with an initial layout that includes recommended tools, workflows, and
-project structure.
+Cookiecutter Fork** command-line utility. It serves as an initial structure
+which can be used by beginners as well as full-fledged developers to simplify
+the project creation process and save considerable amount of time. Cookieninja
```
Comment (xmnlab) [inline docs/index.md:15]:
```suggestion
the project creation process and save considerable amount of time. SciCookie
```

**Review by xmnlab — CHANGES_REQUESTED** [review]
thanks for working on that @Satarupa22-SD !
just added a few notes

**Comment by Satarupa22-SD** [comment]
@xmnlab it's done :)

File: docs/index.md [hunk 3]
```diff
@@ -10,11 +10,11 @@
...
 project templates and is based on
 [Cookieninja](https://github.com/cookieninja-generator/cookieninja) **A
-Cookiecutter Fork** command-line utility. It serves as a boilerplate which can
-be used by beginners as well as full fledged developers to simplify the project
-creation process and save considerable amount of time. Cookieninja enables
-projects with an initial layout that includes recommended tools, workflows, and
-project structure.
+Cookiecutter Fork** command-line utility. It serves as an initial structure
+which can be used by beginners as well as full-fledged developers to simplify
+the project creation process and save considerable amount of time. Scicookie
```
Comment (xmnlab) [inline docs/index.md:15]:
```suggestion
the project creation process and save considerable amount of time. SciCookie
```

File: README.md [hunk 4]
```diff
@@ -10,11 +10,11 @@
...
 project templates and is based on
 [Cookieninja](https://github.com/cookieninja-generator/cookieninja) **A
-Cookiecutter Fork** command-line utility. It serves as a boilerplate which can
-be used by beginners as well as full fledged developers to simplify the project
-creation process and save considerable amount of time. Cookieninja enables
-projects with an initial layout that includes recommended tools, workflows, and
-project structure.
+Cookiecutter Fork** command-line utility. It serves as an initial structure
+which can be used by beginners as well as full-fledged developers to simplify
+the project creation process and save considerable amount of time. Scicookie
```
Comment (xmnlab) [inline README.md:15]:
```suggestion
the project creation process and save considerable amount of time. SciCookie
```

**Review by xmnlab — CHANGES_REQUESTED** [review]

**Review by xmnlab — COMMENTED** [review]
@Satarupa22-SD ,
Just added a small note about the project name, it should `SciCookie`

**Review by xmnlab — APPROVED** [review]
LGTM! Thanks @Satarupa22-SD for working on that

**Pull request merged**